ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-devel__downstream-candidate.yaml

```diff
@@ -25,14 +25,35 @@ tests:
     cluster_profile: azure-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
-      CATALOG_SOURCE_IMAGE: quay.io/redhat-user-workloads/ose-osc-tenant/osc-test-fbc:osc-test-fbc-on-push-ndd8k-build-image-index
+      CATALOG_SOURCE_IMAGE: quay.io/redhat-user-workloads/ose-osc-tenant/osc-test-fbc:osc-test-fbc-on-push-pzw5x-build-image-index
       CATALOG_SOURCE_NAME: brew-catalog
+      ENABLEPEERPODS: "false"
       EXPECTED_OPERATOR_VERSION: 1.10.0
       INSTALL_KATA_RPM: "true"
-      KATA_RPM_BUILD_MD5SUM: 1d108ef4451ce8754eeea08cf12aea0a
-      KATA_RPM_BUILD_TASK: "67952818"
-      KATA_RPM_VERSION: 3.17.0-1.rhaos4.18.el9_6
-      SLEEP_DURATION: "0"
+      KATA_RPM_BUILD_MD5SUM: 65fe515fce039a1ce98aa51dc73655fb
+      KATA_RPM_BUILD_TASK: "3726592"
```

Comment:
if you are using a non-scratch build then you should omit the build task id. It is getting a rpm that doesn't match the md5sum (65fe515fce039a1ce98aa51dc73655fb)